build-system/server/new-server/transforms/transform.ts

```diff
@@ -22,6 +22,7 @@ import transformScriptPaths from './scripts/scripts-transform';
 import transformStories from './stories/stories-transform';
 import transformCss from './css/css-transform';
 
+
```

Comment:
Consider removing since this file apparently doesn't need to be a part of this PR anymore

Reply:
done! thanks!